packages/backend/src/graphql/middleware/index.test.ts

```diff
@@ -125,50 +163,7 @@ describe('GraphQL Middleware', (): void => {
         id: initialResponse.asset.id,
         code: initialResponse.asset.code,
         scale: initialResponse.asset.scale,
-        withdrawalThreshold: null
-      })
-    })
-
-    test('does not return original response on repeat call with different idempotency key', async (): Promise<void> => {
```

Comment:
It didn't make sense to me to keep this test. This PR adds GraphQL Errors that are thrown if a request is made to create something that already exists, so idempotency wouldn't affect cases like that.